recipe/meta.yaml

```diff
@@ -77,53 +80,77 @@ outputs:
-        - test -f ${PREFIX}/include/omnisci/Shared/InlineNullValues.h
-        - test -f ${PREFIX}/include/omnisci/Logger/Logger.h
+        - test -f ${PREFIX}/include/omnisci/Shared/funcannotations.h  # [linux64]
+        - test -f ${PREFIX}/include/omnisci/Shared/InlineNullValues.h  # [linux64]
+        - test -f ${PREFIX}/include/omnisci/Logger/Logger.h  # [linux64]
+        - if not exist %PREFIX%\Shared\funcannotations.h exit 1  # [win]
+        - if not exist %PREFIX%\Shared\InlineNullValues.h exit 1  # [win]
+        - if not exist %PREFIX%\Logger\Logger.h exit 1  # [win]
         # QE
-        - test -f ${PREFIX}/include/omnisci/QueryEngine/OmniSciTypes.h
-        - test -f ${PREFIX}/{{ install_base }}/QueryEngine/RuntimeFunctions.bc
-        - test -f ${PREFIX}/{{ install_base }}/QueryEngine/RuntimeFunctions.bc
-        - test -f ${PREFIX}/{{ install_base }}/QueryEngine/ExtensionFunctions.ast
+        - test -f ${PREFIX}/include/omnisci/QueryEngine/OmniSciTypes.h  # [linux64]
+        - test -f ${PREFIX}/{{ install_base }}/QueryEngine/RuntimeFunctions.bc  # [linux64]
+        - test -f ${PREFIX}/{{ install_base }}/QueryEngine/GeosRuntime.bc  # [linux64]
+        - test -f ${PREFIX}/{{ install_base }}/QueryEngine/ExtensionFunctions.ast  # [linux64]
+        - if not exist %PREFIX%\QueryEngine\OmniSciTypes.h exit 1  # [win]
+        - if not exist %PREFIX%\QueryEngine\RuntimeFunctions.bc exit 1  # [win]
+        - if not exist %PREFIX%\QueryEngine\GeosRuntime.bc exit 1  # [win]
+        - if not exist %PREFIX%\QueryEngine\ExtensionFunctions.ast exit 1  # [win]
         # jar
-        - test -f ${PREFIX}/{{ install_base }}/bin/omnisci-utility-{{ base_version }}.jar
-        - test -f ${PREFIX}/{{ install_base }}/bin/omnisci-jdbc-{{ base_version }}.jar
-        - test -f ${PREFIX}/{{ install_base }}/bin/calcite-1.0-SNAPSHOT-jar-with-dependencies.jar
+        - test -f ${PREFIX}/{{ install_base }}/bin/omnisci-utility-{{ base_version }}.jar  # [linux64]
+        - test -f ${PREFIX}/{{ install_base }}/bin/omnisci-jdbc-{{ base_version }}.jar  # [linux64]
+        - test -f ${PREFIX}/{{ install_base }}/bin/calcite-1.0-SNAPSHOT-jar-with-dependencies.jar  # [linux64]
+        - if not exist %PREFIX%\bin\omnisci-utility-{{ base_version }}.jar exit 1  # [win]
+        - if not exist %PREFIX%\bin\omnisci-jdbc-{{ base_version }}.jar exit 1  # [win]
+        - if not exist %PREFIX%\bin\calcite-1.0-SNAPSHOT-jar-with-dependencies.jar exit 1  # [win]
         # Unspecified
-        - test -f ${PREFIX}/{{ install_base }}/bin/startomnisci
-        - test -f ${PREFIX}/{{ install_base }}/bin/omnisci_insert_sample_data
+        - test -f ${PREFIX}/{{ install_base }}/bin/startomnisci  # [linux64]
+        - test -f ${PREFIX}/{{ install_base }}/bin/omnisci_insert_sample_data  # [linux64]
+        # - if not exist %PREFIX%\bin\startomnisci.exe exit 1  # [win]
+        # - if not exist %PREFIX%\bin\omnisci_insert_sample_data.exe exit 1  # [win]
```

Comment:
`startomnisci` and `omnisci_insert_sample_data` are `bash` scripts. I guess OmniSci should provide the corresponding `.bat` files.